### PR TITLE
add pipeline

### PR DESCRIPTION
**Pull request by LauPerlaza**

### CODE REVIEW

File: .github/jobs/ducks-backend.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+ENVIRONMENT=${ENVIRONMENT:-"develop"}
+
+BUCKET_NAME="web-ducks-server-$ENVIRONMENT"
```
Comment (danielrive) [inline .github/jobs/ducks-backend.sh]: change the name to something that indicate that is a bucket for terrraform state

something like tf-state-ducks...

File: .github/workflows/atomate-terraform.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  set-up-terraform-backend:
+    runs-on: ubuntu-latest
+        
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+              
+      - name: Configure AWS Credentials
+        id: config_aws_creds
+        uses: aws-actions/configure-aws-credentials@v2
```
Comment (danielrive) [inline .github/workflows/atomate-terraform.yml:22]: a new version of this action has been released, please check, i think that is v3 the last one

File: backend.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+terraform {
+
+  backend "s3" {
+    bucket = "web-ducks-server-develop"
```
Comment (danielrive) [inline backend.tf]: this should be a variable

File: backend.tf [hunk 4]
```diff
@@ -0,0 +1,10 @@
+terraform {
+
+  backend "s3" {
+    bucket = "web-ducks-server-develop"
+    key = "web-ducks-server-develop"
```
Comment (danielrive) [inline backend.tf]: same here

File: backend.tf [hunk 5]
```diff
@@ -0,0 +1,10 @@
+terraform {
+
+  backend "s3" {
+    bucket = "web-ducks-server-develop"
+    key = "web-ducks-server-develop"
+    region = "us-east-1"
+    dynamodb_table = "dynamodb-ducks-develop"
```
Comment (danielrive) [inline backend.tf]: same here

**Review by danielrive — CHANGES_REQUESTED** [review]
check comments

File: .github/jobs/ducks-backend.sh (hunk 1 above)
Comment (LauPerlaza) [reply .github/jobs/ducks-backend.sh]: ok

File: .github/workflows/atomate-terraform.yml (hunk 2 above)
Comment (LauPerlaza) [reply .github/workflows/atomate-terraform.yml:22]: Yes, there is a new one that has been pending to be explained for a long time and it was with the use of roles.
I have this link: https://aws.amazon.com/es/blogs/security/use-iam-roles-to-connect-github-actions-to-actions-in-aws/
but can you explain to me please

Comment (danielrive) [reply .github/workflows/atomate-terraform.yml:22]: ok

**Review by danielrive — APPROVED** [review]
LGFM

File: backend.tf (hunk 3 above)
Comment (LauPerlaza) [reply backend.tf]: ok

File: .github/workflows/atomate-terraform.yml (hunk 2 above)
Comment (LauPerlaza) [reply .github/workflows/atomate-terraform.yml:22]: You approved it but this part is missing.

**Review by danielrive — APPROVED** [review]

**Pull request merged**